dahu/core/app/scripts/controller/layout.js

```diff
@@ -144,6 +147,19 @@ define([
                 screencast: this.screencast,
                 screenId: screenId
             }));
+            
+            if(this.currentNoteView === null) {
```

Comment:
currentNoteView is not meant to be accessible from outside the controller. The convention is to prefix the variable name with _